src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp

```diff
@@ -611,6 +612,23 @@ TEST_P(OVClassSetExecutionModeHintConfigTest, SetConfigNoThrow) {
     ASSERT_EQ(ov::hint::ExecutionMode::PERFORMANCE, ie.get_property(target_device, ov::hint::execution_mode));
 }
 
+TEST_P(OVClassSetSchedulingCoreTypeHintConfigTest, SetConfigNoThrow) {
+    ov::Core ie = createCoreWithTemplate();
+
+    OV_ASSERT_PROPERTY_SUPPORTED(ov::hint::scheduling_core_type);
+
+    ov::hint::SchedulingCoreType defaultMode{};
+    ASSERT_NO_THROW(defaultMode = ie.get_property(target_device, ov::hint::scheduling_core_type));
```

Comment:
Shouldn't we also check that default value is actually SchedulingCoreType::ANY_CORE?